docs/dev/setup.rst

```diff
@@ -42,7 +42,8 @@ following dependencies/tools on your machine:
 
    * `GitLab and Jenkins <#jenkins-and-gitlab-setup>`__
    * `GitLab and GitLab CI <#gitlab-ci-and-gitlab-setup>`__ (experimental, not yet production ready)
-   * `Bamboo, Bitbucket and Jira <#bamboo-bitbucket-and-jira-setup>`__)
+   * `Bamboo, Bitbucket and Jira <#bamboo-bitbucket-and-jira-setup>`__
+   * `Local CI and local VC <#local-ci-and-local-vc-setup>`__ (experimental, not yet production ready))
```

Comment:
Removed the bracket.
Yes, the plan is absolutely to have this production ready at some point. There is a [list of missing features](https://confluence.ase.in.tum.de/pages/viewpage.action?pageId=157436217) that need to be implemented to make the system as capable as Bamboo for Artemis purposes. We would also have to do some load testing for big courses but then this might be a viable alternative.